components/buildless-serverless/internal/controller/orphaned-resources/delete_orphaned_resources.go

```diff
@@ -82,6 +82,7 @@ func DeleteOrphanedResources(ctx context.Context, m manager.Manager) error {
 
 	// delete orphaned secrets
 	for _, secret := range secrets.Items {
+		m.GetLogger().Info("Found secret", "name", secret.Name, "namespace", secret.Namespace, "labels", secret.Labels)
```

Comment:
It is inconsistent with previous resources where we have no such log message.
IMO, the message is too general - we don't know why the controller "found secret".

Reply:
Thanks for that, I forgot to remove this debug log